### PR TITLE
docs(file-uploader): usage content updates

### LINKED ISSUE #1403: docs: Update file uploader to follow new component template
Update and improve the usage documentation for file uploader. Follow new component documentation template.

Component template details here: https://www.notion.so/carbondesignsystem/7a1a698e3ad544b1a4180e29febc0cd2?v=bcad0aa949cb4ce1931a71b3be24a771

### PR DESCRIPTION
**Pull request by laurenmrice**
Closes #1403 

Add new images and update content with the new content template

New
- images
- content

### CODE REVIEW

File: src/pages/components/file-uploader/usage.mdx [hunk 1]
```diff
@@ -138,46 +142,250 @@ File uploader allows the user to transfer a file or submit content of their own.
...
+#### Keyboard
+
+####Upload button or drop zone:
+
+- The file uploader button or drop zone can be activated by pressing `Space` or
+  `Enter`.
+- The file uploader button or drop zone should have text or a label that
+  accurately describes the action that will be taken.
+- After the file selection dialog closes, focus should retain the user’s point
+  of rgard and return to the element that invoked it.
```
Comment (jillianhowarth) [inline src/pages/components/file-uploader/usage.mdx:301]: I think there's a typo here

File: src/pages/components/file-uploader/usage.mdx [hunk 2]
```diff
@@ -138,46 +142,250 @@ File uploader allows the user to transfer a file or submit content of their own.
...
+
+####Uploaded file:
+
+- Pressing `Tab` shifts focus to the “x” icon.
+- Pressing `Space` or `Enter` while the "x" icon is in focus will delete the
+  file.
+
+#### Screen readers
```
[the remark this answers is not in the document]
Comment (laurenmrice) [reply src/pages/components/file-uploader/usage.mdx:309]: Right, I wanted to get guidance from a dev on this 👍🏻

File: src/pages/components/file-uploader/usage.mdx [hunk 3]
```diff
@@ -138,46 +142,250 @@ File uploader allows the user to transfer a file or submit content of their own.
...
+
+#### Uploaded file:
+
+- Pressing `Tab` shifts focus to the “x” icon.
+- Pressing `Space` or `Enter` while the "x" icon is in focus will delete the
+  file.
+
+#### Screen readers
+
+- VoiceOver: Users can upload a file by pressing `Space` while the button or
```
Comment (dakahn) [inline src/pages/components/file-uploader/usage.mdx:311]:
```suggestion
- VoiceOver: Users can upload a file by pressing `VO+Space` while the button or
```

File: src/pages/components/file-uploader/usage.mdx [hunk 4]
```diff
@@ -138,46 +142,250 @@ File uploader allows the user to transfer a file or submit content of their own.
...
+  drop zone area is in focus.
+- JAWS: Users can upload a file by pressing `Space` while the button or drop
```
Comment (dakahn) [inline src/pages/components/file-uploader/usage.mdx:313]:
```suggestion
- JAWS: Users can upload a file by pressing `Enter` or `Space` while the button or drop
```

File: src/pages/components/file-uploader/usage.mdx [hunk 5]
```diff
@@ -138,46 +142,250 @@ File uploader allows the user to transfer a file or submit content of their own.
...
+  zone area is in focus.
+- NVDA: Users can upload a file by pressing `Space` while the button or drop
```
Comment (dakahn) [inline src/pages/components/file-uploader/usage.mdx:315]:
```suggestion
- NVDA: Users can upload a file by pressing `Enter` or `Space` while the button or drop
```

**Pull request merged**